koptional/src/main/kotlin/com/gojuno/koptional/Optional.kt

```diff
@@ -6,6 +6,8 @@ sealed class Optional<out T : Any> {
         is Some -> value
         is None -> null
     }
+
+    open operator fun component1() = toNullable()
```

Comment:
Does it need to be open?

Reply:
Otherwise it won't compile because `Some` also declares a `component1`